wodles/aws/aws_s3.py

```diff
@@ -2093,7 +2093,14 @@ def load_information_from_file(self, log_key):
                 "alpn_fe_protocol", "alpn_client_preference_list")
             tsv_file = csv.DictReader(f, fieldnames=fieldnames, delimiter=' ')
 
-            return [dict(x, source='nlb') for x in tsv_file]
+            tsv_file = [dict(x, source='nlb') for x in tsv_file]
```

Comment:
Probable errors are not handled correctly. What if client_port or destination_port are not part of the tsv_file dict?